### ISSUE TITLE
🐞: Switchboard crashes on launching sound plugin.

### ISSUE THREAD

**Issue by joe733** [elementary/switchboard-plug-sound#151]
## Prerequisites
- [X] I have searched open and closed issues for duplicates.

## Describe the bug
I was trying this: [Realtime noise-removal with PulseAudio?](https://askubuntu.com/questions/18958/realtime-noise-removal-with-pulseaudio). But switchboard crashes on launching sound plugin, after changes made in PulseAudio configurations. 

### To Reproduce
Steps to reproduce the behaviour:
1. `$ sudo nano /etc/pulse/default.pa`
2. Add the line: `load-module module-echo-cancel`. Save.
3. Reload PulseAudio `$ pulseaudio -k`
4. Launch sound plugin.

### Expected behaviour
The switchboard does not crash.

### Actual behaviour
- Switchboard crashes after I click sound plugin.
- Sometimes switchboard crashes directly with segmentation fault.
- Sound is gone.
- Reverting the process and reloading PulseAudio with `$ pulseaudio -k` does not work.

After reverting logout and login, then everything works.

### Screenshots or screen recordings
![image](https://user-images.githubusercontent.com/38415384/84735982-6f0f2080-afc2-11ea-9a5d-4c925e2d5c38.png)

### Logs
After I click sound-plugin
```bash             
joe in ~ at eOS via 🅒 base
⮚.⮚.⮚ io.elementary.switchboard       

(io.elementary.switchboard:17710): Gtk-CRITICAL **: 11:13:32.991: gtk_entry_get_text: assertion 'GTK_IS_ENTRY (entry)' failed
[1]    17710 segmentation fault  io.elementary.switchboard
```

## Platform Information
![image](https://user-images.githubusercontent.com/38415384/84736305-2dcb4080-afc3-11ea-8dab-216d9dd9c55a.png)

- [ ] I'm using the latest version from git that I've manually compiled
- [X] I'm using the latest released stable version

**Comment by dbustosrc** [comment]
Hi
I have the same problem
I enable the echo cancelling module in Pulse Audio "module-echo-cancel" and the sound configuration in switchboard doesn't load anymore.
Terminal message:
Segmentation fault

tail -f /var/log/syslog:
Jul 1 14:27:00 dbustos-laptop kernel: [ 1982.946444] io.elementary.s[17890]: segfault at 8 ip 00007f1cea479102 sp 00007ffc535b1dd0 error 4 in libsound.so[7f1cea46b000+1f000]
Jul 1 14:27:00 dbustos-laptop kernel: [ 1982.946454] Code: c5 08 48 8b 48 08 4c 8b 00 31 c0 e8 58 9c ff ff 49 39 ee 75 db 48 8b 83 88 01 00 00 48 8d 15 75 9d 00 00 be 80 00 00 00 31 ff <48> 8b 48 08 4c 8b 00 31 c0 e8 30 9c ff ff 49 8b 44 24 18 48 8b 78

I use pavucontrol too, and it is correct, I have the microphone with echo cancelled.

Any solution to this problem?

**Comment by 4jNsY6fCVqZv** [comment]
Probably a duplicate of https://github.com/elementary/switchboard-plug-sound/issues/122.
Then there will probably be a solution soon :)

**Comment by 4jNsY6fCVqZv** [comment]
Duplicate of #122?
Then there will probably be a solution soon :) #153

**Comment by ThatGeekZen** [comment]
Also having this issue. I have never used pulseaudio's echo cancel. I installed a couple apps like InteliJ from the snap and ran an update which consisted mostly of Mesa driver updates. It was working fine then after a reboot it crashes each time as well. Similarly the same terminal output. Segmentation Fault.  Early this year when I was using eOS I encountered this and reinstalled. Again, I'm not sure whats causing it...
